### PR TITLE
 javascript-javascript1- week4

### PR DESCRIPTION
**Pull request by chunmeishui**
finished javascript week 4 homework

### CODE REVIEW

File: javascript/javascript1/week3/index.js [hunk 1]
```diff
@@ -0,0 +1,253 @@
+//question 1:Item array removal------
```
Comment (aina21) [inline javascript/javascript1/week3/index.js]: remove unrelated file from this PR.

File: javascript/javascript1/week4/index.js [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+    //get the command.name from command
+    let splitt = command.split(` `);
+    if (command.includes('my name is')) {
+        newName.push(splitt);
+        let result = "";
+        for (let name of newName) {
+            result = name;
+            inputName = result[result.length - 1];
+        }
+        function userName(a) {
```
Comment (aina21) [inline javascript/javascript1/week4/index.js]: instead of function you can write like:
```
console.log("Nice to meet you " + inputName)
```

File: javascript/javascript1/week4/index.js [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+            let result = "Nice to meet you " + inputName;
+            console.log(result);
+        } userName(inputName);
+    }
+    else if (command.includes("what is your name")) {
+        newName.push(splitt);
+        function userName(a) {
+            let result = "My name is " + inputName;
+            console.log(result);
+        } userName(inputName);
```
Comment (aina21) [inline javascript/javascript1/week4/index.js]: same here 
`console.log("My name is " + inputName)`

**Review by aina21 — CHANGES_REQUESTED** [review]
good job :+1:

File: javascript/javascript1/week3/index.js (hunk 1 above)
Comment (chunmeishui) [reply javascript/javascript1/week3/index.js]: how to remove this? thanks for telling.

Comment (chunmeishui) [reply javascript/javascript1/week3/index.js]: Hi aina, how can I remove this?

File: javascript/javascript1/week4/index.js (hunk 2 above)
Comment (chunmeishui) [reply javascript/javascript1/week4/index.js]: yes,you are right.That is much easier.

File: javascript/javascript1/week3/index.js (hunk 1 above)
Comment (mariehoff) [reply javascript/javascript1/week3/index.js]: Chunmei, try to ask Ida (aina) on slack instead or just ask in your class channel, to get help with your question
